_post201662016-6-26-rozhovor-vorlicky.md

```diff
@@ -0,0 +1,215 @@
+---
+title: „Společnost se mění k lepšímu. Takže i fotbal, ale bude to strašně dlouho trvat.“
+cover-photo: http://i.imgur.com/MGR7gQLl.jpg
+author: Tomáš Trumpeš
+category: rozhovory
+published: false
```

Comment:
Sem by se ještě hodilo doplnit témata. Něco à la:

    tags: sport, fotbal

Ale není to nutné.